client/ayon_houdini/plugins/publish/collect_componentbuilder_lop.py

```diff
@@ -30,19 +30,22 @@ def process(self, instance):
 
         node = hou.node(instance.data["instance_node"])
 
+        try:
+            node.cook(force=True)  # required to clear existing errors
+        except hou.Error as exc:
+            errors = node.errors()
```

Comment:
This technically is incorrect as far as I know. As far as I'm aware a node _may_ have errors even though the cook itself succeeded.

As such, the `node.errors()` check should actually (or STILL) be done after the `node.parm("execute").pressButton()` ;)

Reply:
https://github.com/ynput/ayon-houdini/pull/193/commits/6513a86567e88c9cd983f9c921c6803b960c34ad